include/cutlass/predicate_vector.h

```diff
@@ -162,6 +165,24 @@ struct PredicateVector {
     bit = byte_offset * 8 + bit_offset + kPredicateStart;
   }
 
+  /// Returns word mask.
+  CUTLASS_HOST_DEVICE static constexpr bool computeWordMask() {
+    Storage mask(0);
+    for (int byte = 0; byte < sizeof(Storage); ++byte) {
+      mask |= (kByteMask << (byte * 8));
+    }
+    return mask;
+  }
+
+  /// Returns mask of last word.
+  CUTLASS_HOST_DEVICE static constexpr bool computeLastWordMask() {
+    Storage mask(0);
+    for (int byte = 0; byte < kBytes % sizeof(Storage); ++byte) {
```

Comment:
all for loops with compile time constant trip count needs `CUTLASS_PRAGMA_UNROLL`

Reply:
This is a constexpr function, I am not sure whether it works well with CUTLASS_PRAGMA_UNROLL. But I will test it locally and submit.